Controller/ResourceController.php

```diff
@@ -39,11 +45,104 @@ public function __construct(
         $this->registry = $registry;
     }
 
-    public function resourceAction($repositoryName, $path)
+    public function getResourceAction($repositoryName, $path)
     {
         $repository = $this->registry->get($repositoryName);
         $resource = $repository->get('/'.$path);
 
+        return $this->createResponse($resource);
+    }
+
+    /**
+     * There are two different changes that can currently be done on a cmf resource:
+     *
+     * - move
+     * - rename
```

Comment:
@dantleech that would be awesome if it fieels equal.

The Comment maybe can be seen as a hint, what to document afterwards, cause we need to explain it.